native/src/main/java/io/ballerina/lib/cdc/utils/Constants.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.lib.cdc.utils;
+
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BString;
+
+/**
+ * This class contains constants used in the CDC module.
+ */
+public class Constants {
```

Comment:
I think we can do the same with an interface, so we don't have to add a private constructor.
```suggestion
public interface Constants {
```

Reply:
"Hmm, is this actually good practice? I thought interfaces were meant to define behavior, like helping different classes implement things differently based on their state."

Reply:
This is something which has different feelings among developer community [1] Anyway we should mark the class as `final`.

[1] - https://vicky-ivanova.medium.com/defining-constants-in-java-interfaces-good-or-bad-practice-e0fecbcd2878